views/trials-details.html

```diff
@@ -78,5 +78,17 @@
       </dd>
     {% endfor %}
   </dl>
+
+  <h2>Publications</h2>
+  <ul>
+    {% for publication in trial.publications %}
+      <dt>
```

Comment:
So, `<dl>` are definition lists. They need a term (`<dt>`) and its value (`<dd>`). We're now just showing a list of publications, we would have only the terms (`<dt>`), which isn't valid in HTML5 (https://www.w3.org/TR/html-markup/dl.html)

This is different from the others, because they all have terms and values. To be honest, I'm probably abusing the `<dl>` element in some parts here (e.g. "Sources") so I might end up changing it to regular `<ul>` lists later. For now, just change publications for `<ul>` and `<li>` :+1: